### PR TITLE
Docs-4154 Update Local Group Articles

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit renames the LocalGroup.md articles to LocalGroupsScreens.md and ManageLocalGroups.md Updates the content and screenshots in the UI ref article to cover the screens and settings, and in the how-to to reflect screen changes. It fixes broken links by renaming the LocalUser.md articles to what they changed to in PR1510.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/Credentials/ManageLocalGroups.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+Click <i class="material-icons" aria-hidden="true" title="Settings">settings</i> **Toggle Built-In Groups** icon again to open the **Hide Built-In Groups** dialog. Click **Hide** to show only non-built-in groups on the system.
+
+## Adding a New Group
+
+To create a group, go to **Credentials > Local Groups** and click **Add**.
+
+![AddGroupSCALE](/images/SCALE/22.02/AddGroupSCALE.png "Add Group") 
+
+Enter a unique number for the group ID in **GID**) that TrueNAS uses to identify a Unix group. Enter a number above 1000 for a group with user accounts or for a system service enter the default port number for the service as the GID.  Enter a name for the group. The group name cannot begin with a hyphen (-) or contain a space, tab, or any of these characters: colon (:), plus (+), ampersand (&), hash (#), percent (%), carat (^), open or close parentheses ( ), exclamation mark (!), at symbol (@), tilde (~), astrick (*), question mark (?) greater or less than (<) (>), equal ). You can only use the dollar sign ($) as the last character in a user name.
```
Comment (tonyriv3) [inline content/SCALE/SCALETutorials/Credentials/ManageLocalGroups.md:33]:
```suggestion
Enter a unique number for the group ID in **GID**) that TrueNAS uses to identify a Unix group. Enter a number above 1000 for a group with user accounts or for a system service enter the default port number for the service as the GID.  Enter a name for the group. The group name cannot begin with a hyphen (-) or contain a space, tab, or any of these characters: colon (:), plus (+), ampersand (&), hash (#), percent (%), carat (^), open or close parentheses ( ), exclamation mark (!), at symbol (@), tilde (~), asterisk (*), question mark (?) greater or less than (<) (>), equal ). You can only use the dollar sign ($) as the last character in a user name.
```

File: content/SCALE/SCALEUIReference/Credentials/LocalGroupsScreens.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+description: "This article provides information on group settings and screens."
+weight: 20
+aliases:
+tags:
+ - scalegroups
+---
+
+{{< toc >}}
+
+The **Credentials > Groups** screen displays a list of groups configured on the screen. By default built-in groups are hidden until you make them visible.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/Credentials/LocalGroupsScreens.md:12]:
```suggestion
The **Credentials > Groups** screen displays a list of groups configured on the screen. By default, built-in groups are hidden until you make them visible. 
```

**Review by tonyriv3 — CHANGES_REQUESTED** [review]

File: content/SCALE/SCALETutorials/Credentials/ManageLocalGroups.md (hunk 1 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/Credentials/ManageLocalGroups.md:33]: oops we have a typo, I'll change that after I commit this suggestion.

File: content/SCALE/SCALETutorials/Credentials/ManageLocalGroups.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Click <i class="material-icons" aria-hidden="true" title="Settings">settings</i> **Toggle Built-In Groups** icon again to open the **Hide Built-In Groups** dialog. Click **Hide** to show only non-built-in groups on the system.
+
+## Adding a New Group
+
+To create a group, go to **Credentials > Local Groups** and click **Add**.
+
+![AddGroupSCALE](/images/SCALE/22.02/AddGroupSCALE.png "Add Group") 
+
+Enter a unique number for the group ID in **GID**) that TrueNAS uses to identify a Unix group. Enter a number above 1000 for a group with user accounts or for a system service enter the default port number for the service as the GID.  Enter a name for the group. The group name cannot begin with a hyphen (-) or contain a space, tab, or any of these characters: colon (:), plus (+), ampersand (&), hash (#), percent (%), carat (^), open or close parentheses ( ), exclamation mark (!), at symbol (@), tilde (~), asterisk (*), question mark (?) greater or less than (<) (>), equal ). You can only use the dollar sign ($) as the last character in a user name.
```
Comment (micjohnson777) [inline content/SCALE/SCALETutorials/Credentials/ManageLocalGroups.md:33]:
```suggestion
Enter a unique number for the group ID in **GID** that TrueNAS uses to identify a Unix group. Enter a number above 1000 for a group with user accounts or for a system service enter the default port number for the service as the GID.  Enter a name for the group. The group name cannot begin with a hyphen (-) or contain a space, tab, or any of these characters: colon (:), plus (+), ampersand (&), hash (#), percent (%), carat (^), open or close parentheses ( ), exclamation mark (!), at symbol (@), tilde (~), asterisk (*), question mark (?) greater or less than (<) (>), equal ). You can only use the dollar sign ($) as the last character in a user name.
```

**Review by tonyriv3 — APPROVED** [review]

**Review by tonyriv3 — APPROVED** [review]

**Pull request merged**